eng/pipelines/templates/jobs/ci.yml

```diff
@@ -159,7 +159,7 @@ jobs:
         inputs:
           command: custom
           custom: 'tool'
-          arguments: 'install --global --add-source "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" --version "1.0.0-dev.20210730.3" "Azure.Sdk.Tools.SnippetGenerator"'
+          arguments: 'install --global --add-source "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" --version "1.0.0-dev.20211119.1" "Azure.Sdk.Tools.SnippetGenerator"'
```

Comment:
The outer quotes are unnecessary here (only when starting with certain characters like `*`).